### DISCUSSION TITLE
Github Pages is not updating my website

### DISCUSSION THREAD

**Discussion by QuintinN** [community/community#76997] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area
Github Pages 

Product Feedback

### Body

Hi, so I used create-react-app a couple years ago to create my own personal website in college. When I tried pushing some changes a year ago it completely broke and I was unable to fix my website. Now, I completly redesigned it and am trying to host on github pages, however pages keeps showing the old broken site. I've created new repositorys and committed code there so I'm just not sure whats going on. Here's my repo. https://github.com/QuintinN/website. Thanks.

**Comment by guspan-tanadi** [comment]
Published to repo on dist directory rather than the source. The topic having related [discussion](https://github.com/vitejs/vite/discussions/10575) of react app.

**Comment by notsointresting** [comment]
It seems like you might be facing a caching issue. GitHub Pages might still be serving the old site from its cache. Here are a few steps you can try to resolve this issue:

1. Clear your browser cache and try again.
2. Try accessing the site from a different browser or in incognito mode.
3. Make sure your new site's code is on the `gh-pages` branch (for project repositories) or on the `master` branch (for user and organization repositories).
4. Check the repository settings to make sure GitHub Pages is being built from the correct branch.
5. Make a small change to your site and push the changes to GitHub. This should trigger a new build and refresh the cache.

If none of these steps work, it might be a good idea to contact GitHub Support for further assistance.
